sycl/doc/developer/DockerBKMs.md

```diff
@@ -40,6 +40,14 @@ The following containers are publicly available for DPC++ compiler development:
    building DPC++ compiler from source.
 - `ghcr.io/intel/llvm/ubuntu2204_intel_drivers`: contains everything from the
    base container + pre-installed Intel drivers.
+   Intel drivers are downloaded from release/tag and saved in dependencies.json.
```

Comment:
The important difference between `ubuntu2204_intel_drivers` and `ubuntu2204_intel_drivers_unstable` which we should highlight is that drivers in `ubuntu2204_intel_drivers` are validated and pass testing, where is drivers installed to `ubuntu2204_intel_drivers_unstable` are not validated.

Reply:
Good point. updated.